common/src/Microsoft.Azure.IIoT.Core/src/TSI/Runtime/TsiConfig.cs

```diff
@@ -0,0 +1,30 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.Core.TSI.Runtime {
+    using Microsoft.Azure.IIoT.Utils;
+    using Microsoft.Extensions.Configuration;
+    using System;
+
+    /// <summary>
+    /// TSI configuration
+    /// </summary>
+    public class TsiConfig : ConfigBase, ITsiConfig {
+
+        private const string kTsi_DataAccessFQDN = "Tsi:DataAccessFQDN";
+
+        /// <summary>DataAccessFQDN</summary>
+        public string DataAccessFQDN => GetStringOrDefault(kTsi_DataAccessFQDN,
```

Comment:
It is nice work!  But my preference would be to just add this property to the App config directly - the platform core should not have anything TSI in it.   If you need to inject an interface, feel free to move both this and above file to the engineering tool codebase.   Leave the PcsVariable entry though.